### PR TITLE
Labels and layout fixes

### PR DESCRIPTION
**Pull request by Bars92**
force height on item when calculating layout height to use overflow height on charts that use autosizer with overflow visible
added percentage view for pie charts and chart legend and removed pie chart percentage labels
added percentage test
removed unneeded spacings
removed showValues injection on bar charts

update no results icon 
image

### CODE REVIEW

File: src/components/Sections/SectionChart/SectionBarChart.js [hunk 1]
```diff
@@ -240,18 +240,10 @@ const SectionBarChart = ({ data, style, dimensions, legend, chartProperties = {}
...
-                    valueAccessor={(entry) => {
-                      let value = '';
-                      if (entry && entry.data && entry.data[0] !== undefined) {
-                        value = entry.data[0];
-                      } else if (entry && entry.value && entry.value[1] !== undefined) {
-                        value = entry.value[1];
-                      }
-                      return value;
-                    }}
+                    dataKey={group => group.total}
```
Comment (yanyo87) [inline src/components/Sections/SectionChart/SectionBarChart.js:246]: i think this should to the job 
```suggestion
                    dataKey={group => group.data[0}
```

File: src/components/Sections/SectionChart/SectionChart.less [hunk 2]
```diff
@@ -3,13 +3,7 @@
...
   width: 100%;
-  display: table;
   .section-title {
     font-size: 18px;
-    margin: 10px 0;
-    display: table-row;
-  }
-  .section-chart-content {
-    display: table-row;
   }
```
Comment (yanyo87) [inline src/components/Sections/SectionChart/SectionChart.less:8]: this causes the bar chart to be outside the widget border 
image


before fix
image

File: src/components/Sections/SectionChart/SectionBarChart.js (hunk 1 above)
Comment (Bars92) [reply src/components/Sections/SectionChart/SectionBarChart.js:246]: i need the total, so i rather make sure its calculated

File: src/components/Sections/SectionChart/SectionPieChart.js [hunk 3]
```diff
@@ -87,7 +57,11 @@ const SectionPieChart = ({ data, style, dimensions, legend, chartProperties = {}
   if (sortBy) {
     preparedData = orderBy(preparedData, sortBy.values, sortBy.orders);
   }
-
+  if (total > 0 && (chartProperties.showValues === undefined || chartProperties.showValues)) {
```
Comment (yanyo87) [inline src/components/Sections/SectionChart/SectionPieChart.js:60]:
```suggestion
  if (total > 0 && (chartProperties.showValues === true )) {
```

File: src/components/Sections/SectionChart/SectionPieChart.js [hunk 4]
```diff
@@ -87,7 +57,11 @@ const SectionPieChart = ({ data, style, dimensions, legend, chartProperties = {}
...
+    preparedData.forEach((group) => {
+      group.percentage = (group.value / total) * 100;
```
Comment (yanyo87) [inline src/components/Sections/SectionChart/SectionPieChart.js:62]:
```suggestion
      group.percentage = (group.value / group.data[0]) * 100;
```

File: src/components/Sections/SectionChart/SectionPieChart.js (hunk 3 above)
Comment (Bars92) [reply src/components/Sections/SectionChart/SectionPieChart.js:60]: i also would like to support backward compatibility (still show percentage labels)

**Review by yanyo87 — APPROVED** [review]

**Pull request merged**